beets/ui/__init__.py

```diff
@@ -1145,9 +1157,8 @@ def _configure(options):
     # Add any additional config files specified with --config. This
     # special handling lets specified plugins get loaded before we
     # finish parsing the command line.
-    if getattr(options, 'config', None) is not None:
-        config_path = options.config
-        del options.config
+    config_path = options.pop('config', None)
+    if config_path is not None:
```

Comment:
@sampsyo : what do you think about the confit patch that made it possible? Should we apply it to master?